db/version_builder.cc

```diff
@@ -529,15 +529,15 @@ class VersionBuilder::Rep {
     auto del_it = del_files.find(file_number);
     if (del_it != del_files.end()) {
       del_files.erase(del_it);
```

Comment:
In the case of deletion followed by addition, who calls `RemoveCurrentStats()` on the deleted file, or is it not needed?

Reply:
It's done by `MaybeAddFile` below (that's the "file appearing in both the base version and the added list" case).

Reply:
Oh ok, understood now, thanks!